diff --git a/node_modules/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/LayoutAnimationsProxy.cpp b/node_modules/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/LayoutAnimationsProxy.cpp
index 8102462..a6f477f 100644
--- a/node_modules/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/LayoutAnimationsProxy.cpp
+++ b/node_modules/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/LayoutAnimationsProxy.cpp
@@ -9,6 +9,8 @@
 #include <set>
 #include <utility>
 
+#define LAYOUT_ANIMATIONS_LOGS (1)
+
 namespace reanimated {
 
 // We never modify the Shadow Tree, we just send some additional
@@ -118,6 +120,7 @@ std::optional<SurfaceId> LayoutAnimationsProxy::endLayoutAnimation(
   layoutAnimations_.erase(tag);
   updateMap.erase(tag);
 
+    return {};
   if (!shouldRemove || !nodeForTag_.contains(tag)) {
     return {};
   }
@@ -127,6 +130,10 @@ std::optional<SurfaceId> LayoutAnimationsProxy::endLayoutAnimation(
   mutationNode->state = DEAD;
   deadNodes.insert(mutationNode);
     
+#ifdef LAYOUT_ANIMATIONS_LOGS
+  LOG(INFO) << "endLayoutAnimation surfaceId" << surfaceId << std::endl;
+#endif
+
   return surfaceId;
 }
 
@@ -151,6 +158,7 @@ void LayoutAnimationsProxy::parseRemoveMutations(
       deletedViews.insert(mutation.oldChildShadowView.tag);
     }
     if (mutation.type == ShadowViewMutation::Remove) {
+
       updateIndexForMutation(mutation);
       auto tag = mutation.oldChildShadowView.tag;
       auto parentTag = mutation.parentShadowView.tag;
@@ -181,6 +189,10 @@ void LayoutAnimationsProxy::parseRemoveMutations(
       }
       nodeForTag_[tag] = mutationNode;
         
+    #ifdef LAYOUT_ANIMATIONS_LOGS
+            LOG(INFO) << "parseRemoveMutations parent" << parentTag << " " << unflattenedParentTag << std::endl;
+    #endif
+
       if (!parent) {
         parent = std::make_shared<Node>(parentTag);
         nodeForTag_[parentTag] = parent;
@@ -454,7 +466,7 @@ bool LayoutAnimationsProxy::startAnimationsRecursively(
     bool isScreenPop,
     ShadowViewMutationList &mutations) const {
   if (isRNSScreen(node)) {
-    isScreenPop = true;
+     isScreenPop = true;
   }
 
   shouldAnimate = !isScreenPop &&
